setup.py

```diff
@@ -861,6 +861,33 @@ def run(self) -> None:
                 target_hipblaslt_dir = target_dir / "hipblaslt"
                 target_hipblaslt_dir.mkdir(parents=True, exist_ok=True)
                 self.copy_tree(hipblaslt_dir, str(target_hipblaslt_dir))
+
+                # Copy libomp140.x86_64.dll to torch/lib since rocalution.dll needs it
+                omp_name = 'libomp140.x86_64.dll'
```

Comment:
As it turns out, OpenMP is also a dependency of `torch_cpu.dll` (when `USE_OPENMP` is set, which it is by default), not just `rocalution.dll`. See https://github.com/ROCm/TheRock/issues/1309.

I've taken some of the ideas from this PR and adapted them into our downstream build script for torch+ROCm packages on Windows in TheRock here: https://github.com/ROCm/TheRock/pull/1520.

Given how core this dep is to torch_cpu, I'm wondering what sort of upstream solution would be best. Some ideas have been proposed at https://github.com/pytorch/pytorch/pull/160258.